bigquery_etl/cli/query.py

```diff
@@ -1479,64 +1479,77 @@ def deploy(
         query_file_path = Path(query_file)
         existing_schema_path = query_file_path.parent / SCHEMA_FILE
 
-        if existing_schema_path.is_file():
-            click.echo(f"Deploy schema for {query_file}")
+        if not existing_schema_path.is_file():
+            click.echo(f"No schema file found for {query_file}")
+            continue
+
+        click.echo(f"Deploying schema for {query_file}")
+
+        table_name = query_file_path.parent.name
+        dataset_name = query_file_path.parent.parent.name
+        project_name = query_file_path.parent.parent.parent.name
+        full_table_id = f"{project_name}.{dataset_name}.{table_name}"
 
-            table_name = query_file_path.parent.name
-            dataset_name = query_file_path.parent.parent.name
-            project_name = query_file_path.parent.parent.parent.name
-            existing_schema = Schema.from_schema_file(existing_schema_path)
-            full_table_id = f"{project_name}.{dataset_name}.{table_name}"
+        existing_schema = Schema.from_schema_file(existing_schema_path)
 
-            if not force:
-                query_schema = Schema.from_query_file(
-                    query_file_path,
-                    use_cloud_function=use_cloud_function,
-                    respect_skip=respect_dryrun_skip,
+        if not force:
+            query_schema = Schema.from_query_file(
+                query_file_path,
+                use_cloud_function=use_cloud_function,
+                respect_skip=respect_dryrun_skip,
+            )
+            if not existing_schema.equal(query_schema):
+                click.echo(
+                    f"Query {query_file_path} does not match "
+                    f"schema in {existing_schema_path}. "
+                    f"To update the local schema file, "
+                    f"run `./bqetl query schema update "
+                    f"{dataset_name}.{table_name}`",
+                    err=True,
                 )
-                if not existing_schema.equal(query_schema):
-                    click.echo(
-                        f"Query {query_file_path} does not match "
-                        f"schema in {existing_schema_path}. "
-                        f"To update the local schema file, "
-                        f"run `./bqetl query schema update "
-                        f"{dataset_name}.{table_name}`",
-                        err=True,
-                    )
-                    sys.exit(1)
+                sys.exit(1)  # TODO: Should this be a continue?
```

Comment:
Looks like it might be a short circuit in the process. No need to proceed forward if an error is detected?